client/src/App.js

```diff
@@ -18,7 +18,7 @@ const App = () => {
 	const [energisers, setEnergisers] = useState([]);
 
 	useEffect(() => {
-		const socket = io("https://cyf-booster.herokuapp.com");
+		const socket = io("https://cyf-booster.herokuapp.com/game");
```

Comment:
If you really want to try this link. I can approve it

Reply:
no, it's an old one I forgot to delete.